docs/baking.md

```diff
@@ -105,125 +101,141 @@ In order to import the snapshot, run the following command:
 sudo -u tezos tezos-node-<network> snapshot import <path to the snapshot file>
 ```
 
-### Starting the node
+## Setting up baker and endorser key
 
-After the snapshot import, you can finally start the node by running:
+Note that account activation from JSON file and baker registering require
+running fully-bootstrapped `tezos-node`. In order to start node service do the following:
 ```
 sudo systemctl start tezos-node-<network>.service
 ```
 
-Note that even after the snapshot import the node can still be out of sync. It may require
-some additional time to completely bootstrap. In order to check whether the node is bootstrapped,
-you can use `tezos-client`:
+Even after the snapshot import the node can still be out of sync. It may require
+some additional time to completely bootstrap. In order to check whether the node
+is bootstrapped and wait in case it isn't, you can use `tezos-client`:
```

Comment:
```suggestion
Even after the snapshot import the node can still be out of sync and may require
some additional time to completely bootstrap.

In order to check whether the node is bootstrapped and wait in case it isn't,
you can use `tezos-client`:
```
I think would be a bit more readable in the rendered doc.